velox/exec/Spiller.cpp

```diff
@@ -670,7 +670,6 @@ SpillStats Spiller::stats() const {
 
 // static
 memory::MemoryPool* Spiller::pool() {
-  static auto pool = memory::addDefaultLeafMemoryPool("_sys.spilling");
-  return pool.get();
+  return memory::spillMemoryPool();
```

Comment:
Curious, how do we ensure that all memory allocations during spilling are using this pool.

Reply:
Currently, spiller use spill memory pool for the memory allocation during the spill processing. This diff is to catch any unexpected use cases in the relevant code path.